### ISSUE TITLE
Scraping post date

### ISSUE THREAD

**Issue by ltoshev** [Makepad-fr/fbjs#65]
Hi all,

great lib works very well, can we add a way to scrape the post date too? I tried to look into the lib inner workings, but cannot figure what is the right selector to extract the date. Else I've wrote some func that will parse out properly the date from string to actual date object.

**Comment by kaanyagci** [comment]
Hi @ltoshev ,

Thank you for your feedback. I'll add this to the roadmap and it'll be included in the next release.

**Comment by ltoshev** [comment]
I was able to actually mod the code, are u ok if I open a pull request. Not sure my selector is good but I was able to get out the date of the post as well. I think the selector may be specific for my geography. But works very well.

**Comment by kaanyagci** [comment]
@ltoshev yes please feel free to open a pull request! I can review it ASAP
